vendor/github.com/hashicorp/raft/net_transport.go

```diff
@@ -177,6 +177,19 @@ func (n *NetworkTransport) Close() error {
 	return nil
 }
 
+func (n *NetworkTransport) Reload() {
+	n.shutdownLock.Lock()
+	defer n.shutdownLock.Unlock()
+
+	if !n.shutdown {
+		close(n.shutdownCh)
+		n.shutdown = true
+	}
+
```

Comment:
Create an issue in the raft library and open a PR for this there and vendor the changes in. Otherwise, this will drift when we update raft.